repl/repl.go

```diff
@@ -1199,6 +1209,7 @@ var builtin = [...]commandDesc{
 	{"profile", []string{}, "toggle profiler and turns off trace"},
 	{"types", []string{}, "toggle type information"},
 	{"unknown", []string{"[ref-1 [ref-2 [...]]]"}, "toggle partial evaluation mode"},
+	{"strict-builtin-errors", []string{}, "treat built-in function errors as fatal"},
```

Comment:
[nit] Shouldn't this say "toggle", too?

Also, is there a way to retrieve the current setting? (Might not be needed if we don't have that on any of the other toggling commands...)

Reply:
Fixed.